exercises/rotational-cipher/src/example/java/RotationalCipher.java

```diff
@@ -0,0 +1,38 @@
+
+public class RotationalCipher {
```

Comment:
Maybe the `shiftKey` could be provided in the constructor and stored as a field?